chapters/methodology/theory.tex

```diff
@@ -144,37 +169,43 @@ \subsubsection{CADIS-$\Omega$}
 particle weights according to their importance. However, CADIS-$\Omega$ will
 use the
 $\Omega$-adjoint scalar flux rather than the standard adjoint scalar flux
-to generate the biased source distribution and the particle weights.
+to generate the biased source distribution, weight windows, and particle birth weights.
 The consistency between the biased source and particle weights is maintained
-when using $\phi_{\Omega}^{\dagger}$ rather than $\phi^{\dagger}$. The adjusted
+when using $\phi_{\Omega}^{\dagger}$ rather than $\phi^{\dagger}$. 
+% I don't understand the above sentence. 
```

Comment:
I'm trying to say that because we're placing the omega flux in both the biased source distribution and to calculate the particle weights, we maintain the consistent-biasing that CADIS is characterized by.